spec/models/github_repository_spec.rb

```diff
@@ -18,31 +18,97 @@
     @client.delete_repository(@github_repository.id)
   end
 
-  describe '#present?', :vcr do
-    it 'returns true if the repo is present' do
-      expect(GitHubRepository.present?(@client, 'rails/rails')).to be_truthy
+  describe 'class methods' do
+    describe '::present?', :vcr do
+      context 'without options' do
+        it 'returns true if the repo is present' do
+          expect(GitHubRepository.present?(@client, 'rails/rails')).to be_truthy
+        end
+
+        it 'returns false if the repo is not present' do
+          expect(GitHubRepository.present?(@client, 'foobar/jim')).to be_falsey
+        end
+      end
+
+      context 'with options' do
```

Comment:
@tarebyte thanks for reviewing! 

I just modified `::present?` and `#present?` based on your feedback.
Also added some tests for the custom options, would you mind having a 👀  at them?